migrations/20190204190814_imgtable.js

```diff
@@ -0,0 +1,14 @@
+
+exports.up = function (knex, Promise) {
+  return knex.schema.createTable('images', function (t) {
+    t.increments().primary();
+    t.string('author').defaultTo('anonymous').notNullable();
```

Comment:
You should create a new migration to remove the `defaultTo` here. Each gallery should be owned by someone. 
You'll also need a foreign key to represent the user from the images table.